common/changes/@uifabric/experiments/ahackett-tilehoverfix96_2018-09-06-19-34.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/experiments",
+      "comment": "Fix on hover underline issue for signal icon in Tiles view",
```

Comment:
Please add the component name in front, e.g. Tiles: Fix...

Reply:
Fixed